task_2.py

```diff
@@ -1 +1,59 @@
-# Remove this comment to confirm that this task is done
\ No newline at end of file
+from turtle import forward, speed, exitonclick,backward, left, pendown, penup, goto
+import random
+speed("normal")
+penup()
+goto(-270,-200)
+pendown()
+def rectangle(x,y):
+    for _ in range(2):
+        backward(x)
+        left(90)
+        forward(y)
+        left (90)
+def gap():
+    penup()
+    forward(30)
+    pendown()
+rectangle(20,50)
+gap()
+rectangle(20,70)
+gap()
+rectangle(20,100)
+gap()
+rectangle(20,130)
+gap()
+rectangle(20,140)
+gap()
+rectangle(20,160)
+gap()
+rectangle(20,150)
+gap()
+rectangle(20,150)
+gap()
+rectangle(20,170)
+gap()
+rectangle(20,175)
+gap()
+rectangle(20,190)
+gap()
+rectangle(20,220)
+gap()
+rectangle(20,240)
+gap()
+rectangle(20,240)
+gap()
+rectangle(20,270)
+gap()
+rectangle(20,290)
+gap()
+rectangle(20,290)
+gap()
+rectangle(20,290)
+gap()
+rectangle(20,310)
+gap()
+rectangle(20,330)
+gap()
+exitonclick()
```

Comment:
![image](https://user-images.githubusercontent.com/56234206/197409027-d2e31c05-4d06-4c25-9901-3158961efd9a.png)

Błagam, zrób to w pętli i po prostu dodawaj za każdym razem wynik wywołania funkcji `randint(a, b)` (a i b musisz sobie sama wymyślić, tak żeby Ci pasowało.